### ISSUE TITLE
It seems like alt + left mouse button selection doesn't work when using this addon.

### ISSUE THREAD

**Issue by mzso** [fanfare/googleimagesrestored#18]
Hello!

It seems like for some reason I can't select text of links via this method when this addon is active.
It works with the unmanipulated layout.

I use this feature quite a lot, so it's rather annoying. Browser is Firefox, didn't try with Chrome.

**Comment by fadster** [comment]
Indeed, I confirm this behavior. Any chance for a fix?

**Comment by fadster** [comment]
Indeed, I confirm this behavior (using Chrome). Any chance for a fix?
